winui/Shadow/Overview.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: What's New for .NET WinUI Shadow Control | Syncfusion
+description: The .NET WinUI Shadow control is used to apply shadow effects to any framework element to create a beautiful and appealing UI.
+platform:winui-controls
+control: SfShadow
+documentation: ug
+---
+
+# WinUI Shadow (New)
+
+The [WinUI Shadow](https://www.syncfusion.com/winui-controls/shadow) .NET WinUI Shadow control is used to apply shadow effects to framework elements to give them an appearance of depth, allowing users to easily differentiate overlapping elements and enjoy a beautiful and appealing user interface.
+
+## Shadow control
+
+![Displaying the Shadow control]()
+
+## Key features
+
+* Shadow Color: Applies the user's own color to the `shadow`.
+* Shadow BlurRadius: Applies a blur radius for the `shadow`.
+* Shadow Offset: `OffsetX` and `OffsetY` properties, position the shadow relative to the position of the view.
+* Shadow CornerRadius: SfShadow provide CornerRadius property, you can soften the `shadow` edges and make them look more essential. 
```

Comment:
@Priyanka-Vijayasankar Applies the corner radius for the shadow.

Reply:
Change the content in corner radius as mentioned above.